README.md

```diff
@@ -30,25 +30,25 @@ This gem is an alternative and inspired by the great work in the other gem [simp
 Whereever you are integrating `SimpleCov` you can configure the `SimpleCovSmallBadge` gem as any formater can be configured. The default integration could looks as follows:
 
 ```ruby
-require 'simple-cov-small-badge'
+require 'simplecov_small_badge'
 
 # Wherever your SimpleCov.start block is (spec_helper.rb, test_helper.rb, or .simplecov)
 SimpleCov.start do
   # add your normal SimpleCov configs
   add_filter "/app/model"
   # call SimpleCov::Formatter::BadgeFormatter after the normal HTMLFormatter
-  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new([
     SimpleCov::Formatter::HTMLFormatter,
-    SimpleCovSimpleBadge::Formatter
-  ]
+    SimpleCovSmallBadge::Formatter
+  ])
```

Comment:
Fixed the typo.
Also, since there is a deprecation warning from SimpleCov: [here](https://github.com/colszowka/simplecov/blob/d4be523b624a68c32e03264437af48d0108ecea1/lib/simplecov/formatter/multi_formatter.rb#L29) I changed the Readme to reflect that.
The change from **formatter** to **formatters** is also referencing the [SimpleCov Doc's](https://github.com/colszowka/simplecov#using-multiple-formatters)